lib/pagem.rb

```diff
@@ -98,3 +98,45 @@ def onkeypress_script
     "if(event.keyCode == 13) Pager.setPage(this, '#{@page_variable}', this.value, #{@is_remote});"
   end
 end
+
+# PagemMultiscope class inherits from Pagem and allows pagination of 2 scopes, returning either the paginated
+# set of the first scope with limit and offset, the paginated set of the second scope with limit and offset,
+# or results of a combined scope proc combining both scopes if the page is rendering a combination
+# of both table records on the same page
+# TODO Move this back to the Pagem class and extend it to allow more than 2 scopes
+class PagemMultiscope < Pagem
+  def initialize(first_scope, second_scope, params, opt = {})
+    super(first_scope, params, opt)
+    @first_scope, @second_scope = first_scope, second_scope
+    @first_scope_count, @second_scope_count = first_scope.count, second_scope.count
+    @count = @first_scope_count + @second_scope_count
```

Comment:
I imagine things would break here if either scope is using a GROUP by (since group.count will return a hash instead of numbers you can add together). Maybe we should add some protection for that? I think the Pagem class gets around it by allowing users to pass in a count_number option. Maybe we can do the same here just so it's a bit more flexible?